R/drive_cp.R

```diff
@@ -0,0 +1,118 @@
+#' Copy a Google Drive file.
+#'
+#' `drive_copy()` is an alias for `drive_cp()`.
+#'
+#' @seealso Wraps the
+#' [drive.files.copy](https://developers.google.com/drive/v3/reference/files/copy)
+#' endpoint
+#'
+#' @template file
+#' @template path
+#' @param name Character, new file name if not specified as part of `path`. Any
+#'   name obtained from `path` overrides this argument. Defaults to "Copy of
+#'   {CURRENT-FILE-NAME}."
+#' @template verbose
+#' @template dribble-return
+#'
+#' @examples
+#' \dontrun{
+#' ## create a file to copy
+#' file <- drive_upload(system.file("DESCRIPTION"), "DESC")
+#'
+#' ## Make a "Copy of" copy in same folder as the original
+#' drive_cp("DESC")
+#'
+#' ## Make an explicitly named copy in same folder as the original
+#' drive_cp("DESC", "DESC-two")
+#'
+#' ## Make an explicitly named copy in a different folder
+#' folder <- drive_mkdir("new-folder")
+#' drive_cp("DESC", folder, "DESC-three")
+#'
+#' ## Behold all of our copies!
+#' drive_search("DESC")
+#'
+#' ## Delete all of our copies and the new folder!
+#' drive_search("DESC") %>% drive_delete()
+#' drive_delete(folder)
+#' }
+#' @export
+drive_cp <- function(file = NULL, path = NULL, name = NULL,  verbose = TRUE) {
```

Comment:
I gave `drive_cp()` the same interface as all of its friends here in the PR. I think you should be able to put the copy in a specific folder and give it a name.